<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentClampForm</class>
 <widget class="QWidget" name="CurrentClampForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <widget class="mold::StatusBarForm" name="statusBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>215</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="mold::RoomHeaderForm" name="header" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>80</height>
    </rect>
   </property>
  </widget>
  <widget class="mold::InvisibleClickableButton" name="myLovelyButton">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>175</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string notr="true">PushButton</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="phase2" native="true">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>110</y>
     <width>75</width>
     <height>60</height>
    </rect>
   </property>
   <widget class="QLabel" name="power2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 16px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string notr="true">1234</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="unitPower2">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>15</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 16px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string notr="true">W</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPhase2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string notr="true">Phase 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="current2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string notr="true">1,5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="unitCurrent2">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>35</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string notr="true">A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="phase3" native="true">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>110</y>
     <width>75</width>
     <height>60</height>
    </rect>
   </property>
   <widget class="QLabel" name="unitPower3">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>15</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 16px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string notr="true">W</string>
    </property>
   </widget>
   <widget class="QLabel" name="current3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string notr="true">1,5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="power3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 16px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string notr="true">1234</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPhase3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string notr="true">Phase 3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="unitCurrent3">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>35</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string notr="true">A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="phase1" native="true">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>110</y>
     <width>75</width>
     <height>60</height>
    </rect>
   </property>
   <widget class="QLabel" name="power1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 16px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string notr="true">1234</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="current1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>30</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string notr="true">1,5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="unitPower1">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>15</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 16px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string notr="true">W</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPhase1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>55</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string notr="true">Phase 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="unitCurrent1">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>35</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 13px;
}</string>
    </property>
    <property name="text">
     <string notr="true">A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="momentary" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>45</y>
     <width>180</width>
     <height>65</height>
    </rect>
   </property>
   <widget class="QLabel" name="powerMomentary">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>20</y>
      <width>65</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 18px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string notr="true">12345</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="currentMomentary">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>35</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 14px;
}</string>
    </property>
    <property name="text">
     <string notr="true">1,5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="unitPowerMomentary">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 18px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string notr="true">W</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMomentary">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 14px;
}</string>
    </property>
    <property name="text">
     <string>Momentary Consumption</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="unitCurrentMomentary">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>10</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 14px;
}</string>
    </property>
    <property name="text">
     <string notr="true">A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="energySince" native="true">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>165</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <widget class="QLabel" name="energy">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 18px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string notr="true">12.345,6</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
   </widget>
   <widget class="QLabel" name="unitEnergy">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>20</y>
      <width>45</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 18px;
font-weight: bold;
}</string>
    </property>
    <property name="text">
     <string notr="true">kWh</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelEnergy">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>230</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family: &quot;Clear Sans Medium&quot;;
font-size: 14px;
}</string>
    </property>
    <property name="text">
     <string>Consumption since dd.mm.yyyy</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="energyReset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>185</y>
     <width>55</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
font-size: 14px;
}</string>
   </property>
   <property name="text">
    <string notr="true">Reset</string>
   </property>
  </widget>
  <zorder>statusBar</zorder>
  <zorder>header</zorder>
  <zorder>phase2</zorder>
  <zorder>phase3</zorder>
  <zorder>phase1</zorder>
  <zorder>momentary</zorder>
  <zorder>energySince</zorder>
  <zorder>energyReset</zorder>
  <zorder>myLovelyButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mold::InvisibleClickableButton</class>
   <extends>QPushButton</extends>
   <header>gui/InvisibleClickableButton.hpp</header>
  </customwidget>
  <customwidget>
   <class>mold::StatusBarForm</class>
   <extends>QWidget</extends>
   <header>gui/StatusBarForm.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>mold::RoomHeaderForm</class>
   <extends>QWidget</extends>
   <header>gui/RoomHeaderForm.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
